build-scripts/fetch-binaries.sh

```diff
@@ -42,4 +42,15 @@ echo $KUBE_VERSION > $KUBE_SNAP_BINS/version
     mv istio-${ISTIO_ERSION}/install/kubernetes/istio-demo-auth.yaml ./istio-yaml/
     mv istio-${ISTIO_ERSION}/install/kubernetes/istio-demo.yaml ./istio-yaml/
   fi
+
+  #Linkerd on Linux
+  if [ "$KUBE_ARCH" = "amd64" ]
```

Comment:
Could you not reuse the if statement used in line 32?
```suggestion
  # Istio and Linkerd is only released on amd64
  if [ "$KUBE_ARCH" = "amd64" ]
```

Reply:
I can combine istio and linkerd in this case.